vector/src/main/res/values/strings.xml

```diff
@@ -727,6 +727,8 @@
     <string name="call_connected">Call connected</string>
     <string name="call_connecting">Call connecting…</string>
     <string name="call_ended">Call ended</string>
+    <string name="missed_audio_call">Missed audio call</string>
+    <string name="missed_video_call">Missed video call</string>
```

Comment:
Create plurals now for future evolution ("3 missed audio calls")?

Reply:
Yes ok